base-notebook/start.sh

```diff
@@ -27,6 +27,17 @@ for f in /usr/local/bin/start-notebook.d/*; do
   esac
   echo
 done
+
+# Modify umask if set with flag
+if [[ -z $NB_UMASK ]]; then
+    echo "Using default umask: ${umask}"
+elif umask $NB_UMASK; then
+    echo "umask $NB_UMASK" >> /home/jovyan/.bashrc
```

Comment:
Ok, I didn't think about that. I will check if setting `os.umask` in the jupyter config is enough, although I don't think this will affect bash terminals launched from Jupyter. Another solution could be do edit a global config file (`/etc/profile` or `/etc/bash.bashrc`). What do you think?

Reply:
:open_mouth: You were right! Jupyter handles everything correctly. Setting `os.umask` is enough.